sql/step_37_suppliers_table.sql

```diff
@@ -10,7 +10,7 @@ CREATE TABLE SUPPLIER (
 	SUP_NOTE VARCHAR(200) NULL,
 	SUP_DELETED char(1) DEFAULT 'N',
 	PRIMARY KEY (SUP_ID)
-);
+) ENGINE=MyISAM;
```

Comment:
I am confused; if I look at load_demo_data.sql I see at line 6176:
`) ENGINE=InnoDB AUTO_INCREMENT=3 DEFAULT CHARSET=utf8mb3 COLLATE=utf8mb3_general_ci;`

I only looked there because I had assumed that the change suggested here should have been made there too.